samples/python/benchmark/bert_benhcmark/README.md

```diff
@@ -0,0 +1,40 @@
+# Bert Benchmark Python* Sample {#openvino_inference_engine_ie_bridges_python_sample_bert_benchmark_README}
+
+This sample demonstrates how to estimate performace of a Bert model using Asynchronous Inference Request API. Unlike [demos](@ref omz_demos) this sample doesn't have configurable command line arguments. Feel free to modify sample's source code to try out different options.
+
+The following Python\* API is used in the application:
+
+| Feature | API | Description |
+| :--- | :--- | :--- |
+| OpenVINO Runtime Version | [openvino.runtime.get_version] | Get Openvino API version |
+| Basic Infer Flow | [openvino.runtime.Core], [openvino.runtime.Core.compile_model] | Common API to do inference: compile a model |
+| Asynchronous Infer | [openvino.runtime.AsyncInferQueue], [openvino.runtime.AsyncInferQueue.start_async], [openvino.runtime.AsyncInferQueue.wait_all] | Do asynchronous inference |
+| Model Operations | [openvino.runtime.CompiledModel.inputs] | Get inputs of a model |
+
+## How It Works
+
+The sample downloads a model and a tokenizer, export the model to onnx, reads the exported model and reshapes it to enforce dynamic inpus shapes, compiles the resulting model, downloads a dataset and runs benhcmarking on the dataset.
+
+You can see the explicit description of
+each sample step at [Integration Steps](../../../../docs/OV_Runtime_UG/integrate_with_your_application.md) section of "Integrate OpenVINO™ Runtime with Your Application" guide.
+
+## Building
+
+To build the sample, please use instructions available at [Build the Sample Applications](../../../../docs/OV_Runtime_UG/Samples_Overview.md) section in OpenVINO™ Toolkit Samples guide.
+
+## Running
+
```

Comment:
This README does not consistent with the rest of Python samples and doesn't contain preparation steps.

Reply:
It doesn't need any prep

Reply:
Updated